### PR TITLE
Pim 5853 behat

### PR DESCRIPTION
**Pull request by willy-ahva**
<3 Thanks for taking the time to contribute! You're awesome! <3

If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/CONTRIBUTING.md 

**Description (for Contributor and Core Developer)**

What does this Pull Request do? reference the related issue?

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       |
| Added Behats                      |
| Changelog updated                 |
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  |
| Tech Doc                          |

### CODE REVIEW

File: features/Context/Page/Base/Grid.php [hunk 1]
```diff
@@ -341,7 +341,7 @@ public function getToolbarCount()
          * If there is no toolbar count, this method
          * should even not be called or should raise a not found exception.
          */
-        if (!$pagination || !$pagination->getText()) {
+        if (!$pagination || !$pagination->getText() || false !== strstr($pagination->getText(), 'null')) {
```
Comment (phaseinducer) [inline features/Context/Page/Base/Grid.php]: :scream:

File: features/Context/Page/Base/Grid.php [hunk 2]
```diff
@@ -806,7 +806,9 @@ public function getColumnHeader($columnName)
      */
     protected function getRows()
     {
-        return $this->getGridContent()->findAll('xpath', '/tr');
+        return $this->spin(function () {
+            return $this->getGridContent()->findAll('xpath', '/tr');
+        }, 'Cannot find xpath "/tr"');
```
Comment (phaseinducer) [inline features/Context/Page/Base/Grid.php]: Too technical imho

File: features/export/product-export-builder/export_products_by_specific_date.feature [hunk 3]
```diff
@@ -49,7 +49,7 @@ Feature: Export products according to a date
     And I press "Save"
     And I should not see the text "There are unsaved changes"
 
-  @javascript @skip # No validation for now
+  @javascript @skip
```
Comment (phaseinducer) [inline features/export/product-export-builder/export_products_by_specific_date.feature]: Maybe add a new tag `@skip-peb`

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Filter/DateTimeFilter.php [hunk 4]
```diff
@@ -126,6 +126,7 @@ protected function addUpdatedSinceLastExport($field, $value)
         }
 
         $lastJobStartTime = $lastCompletedJobExecution->getStartTime();
+        $lastJobStartTime->setTimezone(new \DateTimeZone('UTC'));
```
Comment (phaseinducer) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Filter/DateTimeFilter.php]: Is it the same in mongo filter?

File: features/Context/Page/Base/Grid.php (hunk 2 above)
Comment (juliensnz) [inline features/Context/Page/Base/Grid.php]: Cannot get the grid line?

File: src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Filter/DateTimeFilter.php (hunk 4 above)
Comment (willy-ahva) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Filter/DateTimeFilter.php]: good catch I forgot mongo :+1:

Comment (willy-ahva) [inline src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Filter/DateTimeFilter.php]: done

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**